operations/locustfile.py

```diff
@@ -2,18 +2,28 @@
 from locust.runners import MasterRunner
 
 
-request_body = None
+demographics_request_body = None
+auth_request_body = None
 
 
 @events.test_start.add_listener
 def test_start(environment):
-    global request_body
+    global demographics_request_body
+    global auth_request_body
     if isinstance(environment.runner, MasterRunner):
         # in a distributed run, the master does not typically need any test data
         return
+
     # read the sample request body for the demographics endpoint
-    with open("e2e/src/test/resources/newborn_patient.json", "r") as request_body_file:
-        request_body = request_body_file.read()
+    with open("e2e/src/test/resources/newborn_patient.json", "r") as f:
+        demographics_request_body = f.read()
+
+    # set up the sample request body for the auth endpoint
+    # using a valid test token found in the mock_credentials directory
+    auth_scope = "report-stream"
+    with open("mock_credentials/report-stream-valid-token.jwt") as f:
+        auth_token = f.read()
+    auth_request_body = {"scope": auth_scope, "client_assertion": auth_token}
```

Comment:
This is in relation to @tjohnson7021 [question](https://github.com/CDCgov/trusted-intermediary/pull/371#issuecomment-1584951995) as to whether we should still be seeing the `ERROR Authentication failed` log.  We're seeing that still because the auth endpoint doesn't take JSON.  It takes a `x-www-form-urlencoded` content type instead.  E.g....

```suggestion
    auth_request_body = f"scope={auth_scope}&client_assertion={auth_token}"
```

There may be a more native way with Locust to send `x-www-form-urlencoded` content than my suggestion, though.